src/ngraph/runtime/cpu/cpu_tensor_view.hpp

```diff
@@ -33,7 +33,13 @@ namespace ngraph
                 CPUTensorView(const ngraph::element::Type& element_type,
                               const Shape& shape,
                               const std::string& name = "external");
-                virtual ~CPUTensorView() override;
+
+                CPUTensorView(const ngraph::element::Type& element_type,
+                              const Shape& shape,
+                              void* memory_pointer,
+                              const std::string& name = "external");
+                
+                virtual ~CPUTensorView();
```

Comment:
Was the override specifier removal intentional?

Reply:
it is a side-effect of merging, will revert back.

Reply:
Thanks. Also, we should probably add a note somewhere in the bridge that alignments are tied to what the nGraph backend (ex. IA) assumes. Currently it's the maximal vector word length but that's not really ideal for all tensors. I'm not sure we'll be changing that assumption anytime soon though.

Reply:
yes, it is the responsibility of API users to use properly aligned mem for optimal performance. We already do this for DO in mxnet (64 byte aligned), we will add comments as needed.